fanet_addresses.csv

```diff
@@ -0,0 +1,3 @@
+mac_address,fanet_address
```

Comment:
I'd suggest we try to limit the content directly in the root of the repo.  It seems like we might do other things with specific devices (like automatic provisioning), so how about a `devices` folder at the top level in which this CSV could live?